courses/free-courses-en.md

```diff
@@ -372,6 +372,7 @@
 ### DevOps
 
 * [DevOps Bootcamp](https://www.youtube.com/playlist?list=PL9gnSGHSqcnoqBXdMwUTRod4Gi3eac2Ak) - Kunal Kushwaha
+* [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Nutani, `tch.:` Karl Krueger, `tch.:` Dwayne Lessner, `tch.:` Gundega Dekena (Udacity)
```

Comment:
```suggestion
* [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Karl Krueger, Dwayne Lessner, Gundega Dekena (Udacity)
```
no, we don't need `tch.` annotations. Sorry for the confusion.

Reply:
@eshellman ohk, have committed the change.